trade-finance-manager-api/src/v1/controllers/send-deal-submit-emails.js

```diff
@@ -62,38 +62,32 @@ const sendMiaAcknowledgement = async (deal) => {
   // get the email address for PIM user
   const { email: pimEmail } = await api.findOneTeam(CONSTANTS.TEAMS.PIM.id);
 
-  let templateId;
```

Comment:
Interesting wonder why there were declared on a function scope when they should have been local scope. Good catch.